packages/plugins/src/components/plugin-area/index.js

```diff
@@ -54,6 +55,12 @@ class PluginArea extends Component {
 		super( ...arguments );
 
 		this.setPlugins = this.setPlugins.bind( this );
+		this.memoizedContext = memoize( ( name, icon ) => {
```

Comment:
While it decreases the number of rerenders it doesn't quite solve the issue when using the concept of `occurrences`.

Reply:
Is this related to ordering, or did we notice a chance to reduce rerenders?

Reply:
It doesn't have any impact on the ordering but prevents excessive rerenders.